_layouts/dataset.html

```diff
@@ -3,15 +3,19 @@
 grid: sidebar
 ---
 
-{% include sidebar.html %}
-
+{% include sidebar.html
+  header = page.title
+  downloadUrl = page.downloadUrl
+  downloadSize = page.downloadSize
+  sidebarLinks = page.sidebarLinks
+  sidebarSecondaryLinksTitle = site.sidebar_datasets_title
+  sidebarSecondaryLinks = site.sidebar_datasets_links
+  citationUrl = page.citationUrl
+  citation = page.citation
+%}
```

Comment:
While currently the sidebar appears mostly for datasets, we'll probably reuse it again, so allow selecting of local fields AND global fields for the two sets of buttons.